dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageView.java

```diff
@@ -0,0 +1,54 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+import com.dotmarketing.portlets.templates.design.bean.TemplateLayout;
+import com.dotmarketing.portlets.templates.model.Template;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * @author Will Ezell
+ * @author Jose Castro
+ * @version 4.2
+ * @since Oct 6, 2017
+ */
+public class PageView implements Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)